core/shared/lib/src/model/cart/cart.dart

```diff
@@ -0,0 +1,13 @@
+
+import 'package:shared/shared.dart';
+
+class Cart{
+  List<Product> productlist;
+  int productCount ;
+  int quantity;
```

Comment:
what is the significance of this quantity?

Reply:
quantity removed